### ISSUE TITLE
shin megami tensei 5 issue report

### ISSUE THREAD

**Issue by gidoly** [yuzu-emu/yuzu#7315]
first minor issues on vulkan
 switch:
 
 ![2021111105035200_s](https://user-images.githubusercontent.com/66776795/141188801-1405b1ca-be72-4eab-868b-d1e472a9b3b4.jpg)
 
 yuzu vulkan, open gl:

https://user-images.githubusercontent.com/66776795/141188918-14375e12-6f3e-4dcb-bcb6-c8aeebccbb09.mp4

as you can see texture corrupted

switch:

![2021111105125900_s](https://user-images.githubusercontent.com/66776795/141189088-85e7bcca-077c-4155-b008-61dcbbeb10cb.jpg)

yuzu vulkan:

![yuzu_4jfuCpVOid](https://user-images.githubusercontent.com/66776795/141189144-7d377eb4-2825-4872-a4a8-cdb3b8f50a1c.jpg)

as you can see yuzu vlk looks less darker

switch:

![2021111105184600_s](https://user-images.githubusercontent.com/66776795/141189728-db053a10-a5a5-47b2-89d0-a7380879dfde.jpg)

yuzu vulkan, open gl:

![yuzu_pzj7QLGk4U](https://user-images.githubusercontent.com/66776795/141190391-b4206d52-5d79-44c7-84dc-dc48e1147eab.jpg)

as you can see onion demon looks dark on far distance

scaling issue:

![yuzu_6indSQIEj1](https://user-images.githubusercontent.com/66776795/141193833-a4fc8c89-1477-4f44-85de-6540081d6e21.jpg)
 
 
 <img width="308" alt="yuzu_mmYYufZ0BI" src="https://user-images.githubusercontent.com/66776795/141193912-d160e5b4-fb5d-4146-81bb-ab6b5f41f312.png">

 shadow not scales

![yuzu_AnPkkr2mfh](https://user-images.githubusercontent.com/66776795/141194155-946bf0ca-1bf2-41b7-bad9-e2840b931237.jpg)

open gl performance tanks with scaling

hardware: 
5800x
16gb
2080s
rx560

log:

[smt5 vk log.txt](https://github.com/yuzu-emu/yuzu/files/7515866/smt5.vk.log.txt)

[smt 5 open gl log.txt](https://github.com/yuzu-emu/yuzu/files/7515910/smt.5.open.gl.log.txt)

**Comment by gidoly** [comment]
user reported there is rdna 2 specific issue

https://imgur.com/a/RWiAhaT

above is yuzu and other one is switch

**Comment by EmpirixDC** [comment]
> 
> 
> user reported there is rdna 2 specific issue
> 
> https://imgur.com/a/RWiAhaT
> 
> above is yuzu and other one is switch
> 
> i"ll see if i can reproduce on nvidia too

This issue is not RDNA2 specific, it's happening on my 1080Ti on Yuzu compatible drivers (471.96, as well as 472.12)

Non-Corrupted:
![yuzu_PdJWsoGdd8](https://user-images.githubusercontent.com/44146832/141217825-dd187dd1-2c95-4f63-9408-cb6f908628ee.jpg)

Corrupted:
![yuzu_68hJTkD8vB](https://user-images.githubusercontent.com/44146832/141217829-cf12dc8f-14a1-4c95-89da-1a53b7b4a4b8.jpg)

Both SS are from Yuzu, so the issue might be random. Corrupted is taken on the latest early access build from the time of posting, Non-Corrupted is from the build as of 9th November.

**Comment by Guerdem** [comment]
Yeah It's not just a RDNA2 problem, all the things mention above it's happing here too.

Vulkan
I5 10400f
GTX 1650 Super - 496.49
Windows 11

Texture Corrupted
![1080p corrupted texture](https://user-images.githubusercontent.com/77809260/141333688-cee94831-bc4a-40c7-8666-e62ef9a02328.png)

Scaling issues - shadow not scales
1X 1080p
![shadow 1080p](https://user-images.githubusercontent.com/77809260/141333895-638e209d-b01c-4f33-911e-a8beb8ae2e55.png)
2X 4K
![Shadow 4K](https://user-images.githubusercontent.com/77809260/141334039-ca242fcf-88cc-42a1-abfd-0e381f61c608.png)

Less darker 1x 1080p
![shadow 3 1080p](https://user-images.githubusercontent.com/77809260/141334294-73056f54-587d-4ee3-bf69-9bb00bd76cb5.png)

It's not all of them but some Demons seems darker on far distance, there's a boss called Nuwa who have the same issue. In the case of "Angel" for example, the color of the cloth changes to red or black sometimes when you're not close to them. 


https://user-images.githubusercontent.com/77809260/141337141-c0d9ae78-1cd1-431a-88a5-bbe60464866a.mp4


![Screenshot_1](https://user-images.githubusercontent.com/77809260/141337819-f7d0b094-d178-49fe-8e53-808bb09763ea.png)
![Screenshot_2](https://user-images.githubusercontent.com/77809260/141337837-932c8ffc-d905-4035-9cf6-b65d51f1828b.png)
And when I was playing my first plathrough, this happened

![Screenshot_3](https://user-images.githubusercontent.com/77809260/141338802-946eca53-d6ce-4b7e-9f70-111e8d1f2970.png)

Switch
![Screenshot_3](https://user-images.githubusercontent.com/77809260/141339128-e2837fce-9e7a-458e-a881-1e39f675a793.png)


Vulkan
I5 10400f
GTX 1650 Super - 496.49
Windows 11

**Comment by NineKain** [comment]
there is texture popping, inconsistent framerate when unlocked (stuttery, might be expected), some slight sound distorsion/crackling on certain effects.

3060ti, 5600x, 472.1, win 10, vulkan, anisotropic 16

**Comment by NintendoDrone** [comment]
https://user-images.githubusercontent.com/94038322/141650673-3780e592-7977-4a85-a94d-021cf2682145.mp4

It has something with distance or view angle.

**Comment by KawaiiDinosaur** [comment]
On Vulkan, the Onmoraki demon appears with wrong colors at a certain distance. When the enemy is closer to the camera, it renders correctly.

Expected behavior on the left side of the image. Yuzu Vulkan on the right.

![Captura de tela 2021-11-17 011302](https://user-images.githubusercontent.com/36821201/142134209-6d99bc85-5d8d-4e7e-8d8f-4ccc11978741.png)

**Comment by IgorEisberg** [comment]
Doesn't look like any of the contributors has anything to say, but I'll just put it here... (tried all kinds of graphics settings, nothing prevents that shadow corruption)
![vs1](https://user-images.githubusercontent.com/8811086/143873117-c1661107-448d-4a27-8b01-22f61af079f6.jpg)
![vs2](https://user-images.githubusercontent.com/8811086/143873124-1c3464ae-7a13-4a75-98fb-c1b0621fd967.jpg)

Other than shadow and texture corruptions and a crash when entering a battle once in a while, the game looks gorgeous and works great...
